docs/future/vite.md

```diff
@@ -80,6 +79,62 @@ A function for assigning addressable routes to [server bundles][server-bundles].
 
 You may also want to enable the `manifest` option since, when server bundles are enabled, it contains mappings between routes and server bundles.
 
+## Cloudflare
+
+To get started with Cloudflare, you can use the [`unstable-vite-cloudflare`][template-vite-cloudflare] template:
+
+```shellscript nonumber
+npx create-remix@latest --template remix-run/remix/templates/unstable-vite-cloudflare
+```
+
+#### Bindings
+
+Bindings for Cloudflare resources can be configured [within `wrangler.toml` for local development][wrangler-toml-bindings] or within the [Cloudflare dashboard for deployments][cloudflare-pages-bindings].
+Then, you can access your bindings via `context.env`.
+For example, with a [KV namespace][cloudflare-kv] bound as `MY_KV`:
+
+```ts filename=app/routes/_index.tsx
+export async function loader({ context }) {
+  const { MY_KV } = context.env;
+  const value = await MY_KV.get("my-key");
+  return json({ value });
+}
+```
+
+#### Vite & Wrangler
+
+There are two ways to run your Cloudflare app locally:
+
+```shellscript nonumber
+# Vite
+remix vite:dev
+
+# Wrangler
+remix vite:build # build app before running wrangler
+wranger pages dev ./build/client
```

Comment:
Shouldn't this be 'wrangler'?

Reply:
It is using `wrangler` via `wrangler pages dev`.

Reply:
Typo `wranger ` instead of `wrangler` 😄

Reply:
Oh now I see it 😅 yep you're right